utils.py

```diff
@@ -188,15 +190,20 @@ def validate_cookies_del(flows: list[dict]) -> None:
 
 
 def flows_to_log_info(message: str, flow_dict: dict[str, list]) -> None:
+    """Log flows, maximun flows in a log is 200"""
+    flows_to_log(log.info, message, flow_dict)
+
+
+def flows_to_log(logger_fun: Callable, message: str, flow_dict: dict[str, list]):
```

Comment:
Great refactoring passing the logger callable. There's still a tech debt to finish removing the `flows_to_log_info` can you nuke it and replace/adapt its old unit tests?

Reply:
Adapted the old tests